Sources/KindeSDK/Auth/Auth.swift

```diff
@@ -51,6 +52,9 @@ public final class Auth {
         guard let params = tokenToParse?.parsedJWT else {
             return nil
         }
+        if !params.keys.contains(key) {
+            os_log("The claimed value of \"%@\" does not existed in your token", log: .default, type: .error, key)
```

Comment:
@trunges21 Just a typo.

Can you pls change the log text to:

"The claim value of \"%@\" does not exist in your token"

Reply:
Yes, I updated